Browser_Actions/Browser_ActionTests.cs

```diff
@@ -141,6 +144,268 @@ public void HeadlessMode()
+            Assert.That(dropdownUrl, Is.EqualTo($"{_mainUrl}/dropdown"));
+            var select = new SelectElement(_driver.FindElement(By.Id("dropdown")));
+            select.SelectByText("Option 1");
+            var selectedOption = select.SelectedOption.Text;
+            Assert.That(selectedOption, Is.EqualTo("Option 1"), "Selected option is not 'Option 1'");
+        }
+
+        [Test]
+        public void CheckboxElement()
+        {
+            var elementCheckboxes = _driver.FindElement(By.XPath("//*[@id='content']//*[text()='Checkboxes']"));
+            elementCheckboxes.Click();
+            var checkboxesUrl = _driver.Url;
+            Assert.That(checkboxesUrl, Is.EqualTo($"{_mainUrl}/checkboxes"));
+            var checkboxSelect = _driver.FindElement(By.XPath("//*[@id='checkboxes']/input[1]"));
+            checkboxSelect.Click();
+            bool isChecked = checkboxSelect.Selected;
+            Assert.That(isChecked, Is.True, "'Checkbox 1' is ot selected");
+        }
+
+        [Test]
+        public void RangeElement()
+        {
+            var elementHorizontalSlider = _driver.FindElement(By.XPath("//*[@id='content']//*[text()='Horizontal Slider']"));
+            elementHorizontalSlider.Click();
+            var elementHorizontalSliderUrl = _driver.Url;
+            Assert.That(elementHorizontalSliderUrl, Is.EqualTo($"{_mainUrl}/horizontal_slider"));
+            var inputRange = _driver.FindElement(By.XPath("//*[@type='range']"));
+            // inputRange.SendKeys(Keys.Home + "5") - this does not work somehow, input stays home value- "0"
+            var js = (IJavaScriptExecutor)_driver;
+            js.ExecuteScript("arguments[0].value = '5';", inputRange);
+            var rangeValue = inputRange.GetAttribute("value");
+            Assert.That(rangeValue, Is.EqualTo("5"), "Range input value is not equal to 5");
+        }
+
+        [Test]
+        public void TextInputElement()
+        {
+            var elementInputs = _driver.FindElement(By.XPath("//*[@id='content']//*[text()='Inputs']"));
+            elementInputs.Click();
+            var InputsUrl = _driver.Url;
+            Assert.That(InputsUrl, Is.EqualTo($"{_mainUrl}/inputs"));
+            var elementInputNumber = _driver.FindElement(By.XPath("//*[@type='number']"));
+            elementInputNumber.SendKeys("1");
+            var inputNumber = elementInputNumber.GetAttribute("value");
+            Assert.That(inputNumber, Is.EqualTo("1"));
+        }
+
+        [Test]
+        public void BasicAuth()
+        {
+            var elementBasicAuth = _driver.FindElement(By.XPath("//*[@id='content']//*[text()='Basic Auth']"));
+            elementBasicAuth.Click();
+            var BasicAuthUrl = _driver.Url;
+            Assert.That(BasicAuthUrl, Is.EqualTo($"{_mainUrl}/basic_auth"));
+
+            var login = "admin";
+            var authWindowLink = "https://" + login + ":" + login + "@" +
+            "the-internet.herokuapp.com/basic_auth";
```

Comment:
I got it, strange task, not for alerts testing) will ask about how expected should it work